bigbluebutton-html5/private/config/settings.yml

```diff
@@ -149,10 +149,6 @@ public:
       enabled: false
       # If true, positions the icon next to the screenshare button, if false positions it where BBB had raisedHand button up to BBB 2.6 (right-hand bottom corner in LRT)
       centered: true
-    reactionsButton:
-      # Enables the new raiseHand icon inside of the reaction menu (introduced in BBB 2.7)
-      # If both reactionsButton and raiseHandActionButton are enabled, reactionsButton takes precedence.
-      enabled: true
```

Comment:
this should be kept as it's for the new Reactions button

Reply:
Done